rust/lance-index/src/vector/pq/storage.rs

```diff
@@ -1062,4 +1108,25 @@ mod tests {
         let dist2 = storage.dist_between(v, u);
         assert_eq!(dist1, dist2);
     }
+
+    #[tokio::test]
+    async fn test_remap_with_extra_column() {
```

Comment:
Is this because some old indices will have this extra column and we need to make sure they are supported?